auth/auth.go

```diff
@@ -58,6 +59,14 @@ func (auth *Authenticator) SetSessionCookieName(cookieName string) {
 	auth.sessionCookieName = cookieName
 }
 
+func (auth *Authenticator) ChannelsPerUser() *uint32 {
+	return auth.channelsPerUser
+}
+
+func (auth *Authenticator) SetChannelsPerUser(channelsPerUser *uint32) {
```

Comment:
The variable and method name make it sound like a limit. Can we change it to be called a warn threshold like the others?

Reply:
I've just refactored it to channelsWarningThreshold.